Postgres/AddServer.swift

```diff
@@ -60,7 +79,16 @@ class AddServerViewController: NSViewController, MainWindowModelConsumer {
 	
 	@IBAction func createServer(_ sender: AnyObject?) {
 		guard self.view.window!.makeFirstResponder(nil) else { NSSound.beep(); return }
-        guard availableBinaries.indices.contains(selectedVersionIdx) else {
+        
+        let isValid = if availableBinaries.indices.contains(selectedVersionIdx) {
+            true
+        } else if FileManager().fileExists(atPath: binPath.appending("/postgres")) {
```

Comment:
This check is a good idea, but we should probably present an error message instead of beeping.

Previously, the only way to trigger this error was if you made a debug build without binaries, and the beep was to avoid a crash. Now the error is much more likely to occur if you mistype the binary path, so we should let the user know that the problem is that no binaries were found at the path.